base/cvd/cuttlefish/host/libs/zip/zip_cc.cc

```diff
@@ -478,6 +478,30 @@ Result<uint64_t> ReadableZip::NumEntries() {
   return entries;
 }
 
+Result<std::string> ReadableZip::EntryName(uint64_t index) {
+  CF_EXPECT(impl_.get());
+  zip_t* raw_zip = CF_EXPECT(impl_->raw_.get());
```

Comment:
nit: this two lines seem to be repeated in several functions, consider encapsulating it in `CF_EXPECT(GetRaw())`.

Reply:
Filed b/458175394